tests/integration/routes/account/delete-account-test.js

```diff
@@ -99,7 +99,15 @@ getServer(function (error, server) {
   })
 
   test('DELETE /session/account?include=profile', {todo: true}, function (t) {
-    t.end()
+    var options = _.defaultsDeep({
+      url: '/session/account?include=profile'
+    }, routeOptions)
+
+    server.inject(options, function (response) {
+      t.is(response.statusCode, 400, 'returns 400 status')
```

Comment:
see the instructios in https://github.com/hoodiehq/camp/issues/77, this should not return a `400` but a `200`